### DISCUSSION TITLE
Tell me your favorite curry! あなたのお気に入りのカレーを教えてください！

### DISCUSSION THREAD

**Discussion by kazuyo000207** [NAIST-SE/AttractingContributors#510] [General]
I would like to know your favorite curry.
あなたのお気に入りのカレーが知りたいです．

Retort curry or any kind of curry.
レトルトカレーでもカレーの種類でもなんでも構いません．

If you help with my project, please add your favorite curry to the List of Hobby.
もし、このプロジェクトにご協力いただけるなら、あなたのお気に入りのカレーを「List of Hobby」に追加してください。

https://github.com/kazuyo000207/AttractingContributors-Curry

**Comment by natsuonorepositories** [comment]
I've added green curry!
Do you like green curry??????

**Reply by kazuyo000207** [reply]
Of course.
